packages/client/index.js

```diff
@@ -198,7 +227,23 @@ function buildCallFunction (baseUrl, path, method, methodMeta, throwOnError, ope
     }
   }
 }
-
+function createErrorResponse(message) {
+  return {
+    statusCode: 500,
+    message
+  }
+}
+function sanitizeContentType(contentType) {
+  if (!contentType) { return false }
+  const split = contentType.split(';')
+  return split[0]
+}
+function checkResponseAgainstSchema(body, schema) {
+  const ajv = new Ajv()
+  const validate = ajv.compile(schema)
+  const valid = validate(body)
```

Comment:
This AJV instance should ideally be cached. Ideally all schemas should be loaded in the instance already as they would have an `$id`, making it significantly faster.